src/MspFcComms.cpp

```diff
@@ -60,6 +60,16 @@ namespace FcComms
     }
 
     // Send the rc commands to the FC using the member array of rc values.
+    FcCommsReturns MspFcComms::getRawRC()
+    {
+        MSP_RC msp_getrawRC;
+        msp_getrawRC.getRc(rc_values); // rc_values array initialized in corresponding .hpp file
```

Comment:
You have to get the rc values after you send the message to retrieve them from the FC.